src/js/pages/TwitterHandleLanding.jsx

```diff
@@ -174,24 +178,34 @@ export default class TwitterHandleLanding extends Component {
     // }
 
     if (kindOfOwner === 'CANDIDATE') {
+      // Is this supposed to be this.props.param.candidate_we_vote_id
       params.candidate_we_vote_id = ownerWeVoteId;
-      return <Candidate candidate_we_vote_id {...this.props} />;
+      return (
+        <Candidate candidate_we_vote_id
```

Comment:
Please put the `candidate_we_vote_id` variable being passed into the Candidate component on its own line, horizontally parallel to `match`.